doc/lib/provisioning.txt

```diff
@@ -89,7 +89,7 @@ No logging, asserts, or diagnostic information. Compiled using the ARM Thumb ins
 </table>
 
 <b>Debug Build</b> <br>
-All logging, asserts, and diagnostic information enabled. Debugging symbols available. Compiled using the ARM Thumb instruction set (`-mthumb`). Values are in <i>bytes</i>.
+All logging, asserts, and diagnostic information enabled. Debugging symbols available (Compiled with `-Og -g` flags). Compiled using the ARM Thumb instruction set (`-mthumb`). Values are in <i>bytes</i>.
```

Comment:
-Og?

Reply:
Yes, the `0g` flag as suggested by GCC. https://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html
It mentions "_If you are not using some other optimization option, consider using -Og (see Optimize Options) with -g._"